cpp-data-aggregation/aggregation_producer.cpp

```diff
@@ -104,7 +121,14 @@ int main(int argc, char* argv[]) {
     std::string s_tensors_per_dataset(argv[2]);
     int tensors_per_dataset = std::stoi(s_tensors_per_dataset);
 
+    std::string tensor_text = "Running aggregate scaling producer test with ";
+    tensor_text += "tensor size of ";
+    tensor_text += std::to_string(n_bytes);
+    tensor_text += " bytes and";
+    tensor_text += std::to_string(tensors_per_dataset);
+    tensor_text += " tensors per dataset.";
```

Comment:
This string is only used inside the conditional, so better to build it inside the conditional

Reply:
That's probably also too long for a log message. Try to keep log messages so they'll fit on a standard screen width (80 characters or so). Recommend breaking this into multiple messages